### PR TITLE
#bg front end structure fix #161408561

### PR DESCRIPTION
**Pull request by ibraheemkabir**
### What does this PR do?
- it implements all feedbacks on UI suggested by the LFA

### Description of Task to be completed?
- The get-started page on the index page is redirected to the directly to the login page.
- The app theme is changed to improve color conformity
- The footer issue is fixed to full width and reduced font-size
- The products page modal to add a new product form has been added.
 
### How should this be manually tested?
UI can be viewed on gh-pages link after merge

### My background context you want to provide?
All changes are made as requested by LFA to improve the application UI.

### What are the relevant pivotal tracker stories?(if applicable)
#161408561

### CODE REVIEW

File: js/attlist.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    toggle.classList.toggle("active");
+    tog.classList.toggle("openNav")
+}
+
+
+toggle.addEventListener("click", toggleModal);
+
+
+  (function(){
+    /**
```
Comment (amarachukwu-agbo) [inline js/attlist.js]: please remove the commented-out code. Why are you using var here instead of const?

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var login = document.getElementById("login");
+
+login.addEventListener("click", () => {
+    location.href = "attendantdash.html";
+});   l
```
Comment (amarachukwu-agbo) [inline js/main.js]: Is there any reason why you have the 'l' here?

File: js/nav.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+
+toggle.addEventListener("click", toggleModal);
+
+const art = document.querySelector('article');
+const nav = document.querySelector('header');
+
+  (function(){
+    /**
+     * jQuery
```
Comment (amarachukwu-agbo) [inline js/nav.js]: Please remove commented-out code

File: js/salesrecord.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const art = document.querySelector('article');
+const nav = document.querySelector('header');
+art.onscroll = () => {
+    console.log(window.art.scrollY);
```
Comment (amarachukwu-agbo) [inline js/salesrecord.js]: Do you need the console statement here?

File: style/main.css [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+}
+.show-modal {
+    opacity: 1;
+    visibility: visible;
+    transform: scale(1.0);
+    transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.main_howto{
+
```
Comment (amarachukwu-agbo) [inline style/main.css]: Please remove this selector if not needed

File: style/main.css [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+  li {
+    /* Pro li um 5 erhöhen */
+    counter-increment: none;
+    
+    position: ;
+    margin-bottom: 14px;
+  }
+  
+  li::before {
+    /* Aktuellen Wert ausgeben mit "No." davor und ":" danach ausgeben */
```
Comment (amarachukwu-agbo) [inline style/main.css]: What does this mean?

File: style/mandash.css [hunk 7]
```diff
@@ -675,4 +708,136 @@ padding-top: 13%;
...
+article {
+  position: relative;
+  margin: 20px;
+  top: 0;
+  height: 100vh;
+  width: unset;
+  bottom: 0;
+  right: 0;
+  overflow: auto;
+  /* border-left: solid #2a3843; */
```
Comment (amarachukwu-agbo) [inline style/mandash.css]: Please remove commented-out code

**Review by amarachukwu-agbo — CHANGES_REQUESTED** [review]

File: js/attlist.js (hunk 1 above)
Comment (ibraheemkabir) [reply js/attlist.js]: made the correcctions

File: js/nav.js (hunk 3 above)
Comment (ibraheemkabir) [reply js/nav.js]: done

File: js/salesrecord.js (hunk 4 above)
Comment (ibraheemkabir) [reply js/salesrecord.js]: no just checking for a return value from that method will delete now

File: style/main.css (hunk 5 above)
Comment (ibraheemkabir) [reply style/main.css]: - done

File: style/mandash.css (hunk 7 above)
Comment (ibraheemkabir) [reply style/mandash.css]: done

File: style/main.css (hunk 6 above)
Comment (ibraheemkabir) [reply style/main.css]: removed css was gottten from codepen

**Pull request merged**